vagrant/install-common.sh

```diff
@@ -80,7 +80,7 @@ go get ${SINGULARITY_CRI_REPO}
 cd ${GOPATH}/src/${SINGULARITY_CRI_REPO} && make && sudo make install
 
 go get ${SINGULARITY_SLURM_OPERATOR_REPO}
-
+cd ${GOPATH}/src/${SINGULARITY_SLURM_OPERATOR_REPO} && git checkout staging
```

Comment:
Do we want this to appear ?

Reply:
why not, for staging ok

Reply:
need to remember to remove when merging in master